modules/react/src/deckgl.js

```diff
@@ -142,6 +123,25 @@ const DeckGL = forwardRef((props, ref) => {
       style: null,
       width: '100%',
       height: '100%',
+      // The Deck's animation loop is independent from React's render cycle, causing potential
+      // synchronization issues. We provide this custom render function to make sure that React
+      // and Deck update on the same schedule.
+      _customRender: redrawReason => {
```

Comment:
This will be called frequently and recreating a new method all the time. Maybe just `delete forwardProps._customRender` and document why?